### PR TITLE
add option to  wrap command

### PR DESCRIPTION
**Pull request by Konfekt**

### CODE REVIEW

File: doc/codi.txt [hunk 1]
```diff
@@ -215,10 +215,12 @@ g:codi#interpreters
              Default value is {}.
 
                                                     *g:codi#environment_flags*
```
Comment (metakirby5) [inline doc/codi.txt]: Please also change this header, and make sure that other reference (e.g. table of contents) are updated.

File: doc/codi.txt [hunk 2]
```diff
@@ -215,10 +215,12 @@ g:codi#interpreters
...
-g:codi#environment_flags
-             A string of environment flags passed to the interpreter.
-             Defaults to 'INPUTRC=/dev/null' to ensure codi work independent
-             of the user's inputrc settings.
+g:codi#command_wrapper
+             A list of commands and parameters passed to the interpreter.
```
Comment (metakirby5) [inline doc/codi.txt]: This is not quite accurate - it's more like the prefix of the command.

**Comment by Konfekt** [comment]
Thanks for reviewing. Hopefully all rough edges have been knocked off.

**Comment by metakirby5** [comment]
Last thing - can you add the section to the table of contents at the very top of the documentation file? Thanks!

**Pull request merged**